src/main/java/nb/kafka/operator/AppConfig.java

```diff
@@ -0,0 +1,92 @@
+package nb.kafka.operator;
+
+import java.util.Map;
+
+public class AppConfig {
+  private String kafkaUrl;
+  private String securityProtocol;
+  private short defaultReplicationFactor;
+  private boolean enableTopicDelete;
+  private boolean enableTopicImport;
+  private boolean enableAclManagement;
+  private String operatorId;
+  private Map<String, String> standardLabels;
+  private String usernamePoolSecretName;
+  private String consumedUsersSecretName;
+  private Map<String, String> standardAclLabels;
+
+  public String getKafkaUrl() {
+    return kafkaUrl;
+  }
+  public void setKafkaUrl(String kafkaUrl) {
+    this.kafkaUrl = kafkaUrl;
+  }
+  public String getSecurityProtocol() {
+    return securityProtocol;
+  }
+  public void setSecurityProtocol(String securityProtocol) {
+    this.securityProtocol = securityProtocol;
+  }
+  public short getDefaultReplicationFactor() {
+    return defaultReplicationFactor;
+  }
+  public void setDefaultReplicationFactor(short defaultReplicationFactor) {
+    this.defaultReplicationFactor = defaultReplicationFactor;
+  }
+  public boolean isEnableTopicDelete() {
```

Comment:
I'd prefer isEnabledTopicDelete to be more in spirit of English - also setEnabled...

Reply:
The Eclipse getter/setter generator is not very english-compliant :)
I'll change that.